test/models/patient_test.rb

```diff
@@ -1038,6 +1038,15 @@ def valid_patient
     laboratory.destroy
     assessment.destroy
 
+    # meets defiition: has positive test result more than 10 days ago and another positive test result less than 10 days ago
+    laboratory_1 = create(:laboratory, patient: patient, result: 'positive', specimen_collection: 11.days.ago)
+    laboratory_2 = create(:laboratory, patient: patient, result: 'positive', specimen_collection: 9.days.ago)
+    assessment = create(:assessment, patient: patient, symptomatic: false, created_at: 8.days.ago)
+    verify_patient_status(patient, :isolation_asymp_non_test_based)
```

Comment:
Is there already an anti-test for this somewhere? As in we test that the patient does not have `:isolation_asymp_non_test_based)` status when the earliest lab was 9 days ago (or less)?

If not, we could just write something like this:
```
laboratory_2 = create(:laboratory, patient: patient, result: 'positive', specimen_collection: 9.days.ago)
assessment = create(:assessment, patient: patient, symptomatic: false, created_at: 8.days.ago)
assert_nil Patient.isolation_asymp_non_test_based.find_by_id(id: patient.id).nil?

laboratory_1 = create(:laboratory, patient: patient, result: 'positive', specimen_collection: 11.days.ago)
verify_patient_status(patient, :isolation_asymp_non_test_based)
```

Reply:
yep, good question, those are all here: https://github.com/SaraAlert/SaraAlert/blob/master/test/models/patient_test.rb#L1413

Reply:
Isn't that just the helper method that checks the patient status are what they should be? I think @jkufro's suggestion would still add a test we don't already have, and would essentially be an updated version of [this "anti-test"](https://github.com/SaraAlert/SaraAlert/pull/749/files#diff-81929b9416a87cb2d9e2ccb196e783d98c922d4b6ef555b7ae2c76bac7df2806L1053-L1059) that we had for the old `isolation_non_reporting` definition. But let me know if I'm just missing something.

Reply:
That helper method actually checks both of those. It checks for the correctness of the `status` method here: https://github.com/SaraAlert/SaraAlert/blob/master/test/models/patient_test.rb#L1438

And it also checks to make sure that the patient only shows up under the correct scope on every line before that. Not only does it check to make sure it doesn't show up under `isolation_asymp_non_test_based` here: https://github.com/SaraAlert/SaraAlert/blob/master/test/models/patient_test.rb#L1428, it also checks to make sure that it doesn't show up under any of the other scopes as well.

This `verify_patient_status` method was designed was designed to test all linelist status related functionality thoroughly

Reply:
Understood, but Justin was asking for a test case with a positive lab under 10 days ago. I just looked and I think the missing piece here is that such a test already exists [here](https://github.com/SaraAlert/SaraAlert/blob/b82ddd6e7e34a4f74e076c8403e58ab9bb8baf58/test/models/patient_test.rb#L1057-L1060). With that, I'm satisfied with the testing, but @jkufro I don't want to speak for you so please confirm as well.

Reply:
ah, sorry I misunderstood because I thought he was referring to this assertion needing to be there:
```
assert_nil Patient.isolation_asymp_non_test_based.find_by_id(id: patient.id)
```